src/ui/layout/tree-item.vue

```diff
@@ -137,8 +132,13 @@ export default {
         }
 
         this.openmct.router.on('change:path', this.highlightIfNavigated);
+    },
+    updated() {
         if (this.emitHeight) {
-            this.$emit('emittedHeight', this.$refs.me);
+            // no reliable way to get the fully rendered height without timeout
+            setTimeout(() => {
+                this.$emit('emittedHeight', this.$refs.me.offsetHeight);
+            }, 250);
```

Comment:
hmm, try following? ref: https://vuejs.org/v2/api/#updated
```
updated() {
  this.$nextTick(function () {
    // Code that will run only after the
    // entire view has been re-rendered
  })
}
```

Reply:
Yeah, that does work actually, I've moved it to a document readystate event of "complete" for both mct-tree and tree-item, that only fires once and gives me the same result, BUT if you guys (@akhenry)think I should move it to update, I can just put in a check to see if it's been done yet inside a nextTick callback in updated.

Reply:
@jvigliotta Leave it in the readstate callback, I'm fine with that.